README.rst

```diff
@@ -104,7 +109,9 @@ SageMaker Inference Toolkit serving stack leverages `Model Server for Apache MXN
             """
             return encoder.encode(prediction, accept)
 
-- A handler service that is executed by the model server:
+2. Implement a handler service that is executed by the model server.
+   (`Here is an example <https://github.com/aws/sagemaker-pytorch-serving-container/blob/master/src/sagemaker_pytorch_serving_container/handler_service.py>`__ of a handler service.)
+   For more information on how to define your ``HANDLER_SERVICE`` file, see `the MMS custom service documentation <https://github.com/awslabs/mxnet-model-server/blob/master/docs/custom_service.md>`__.
 
 .. code:: python
```

Comment:
ll. 120-121 could be combined into one line - we're not linting this code, and I don't think it's essential to stick to the 100-character line limit here